og.module

```diff
@@ -86,6 +86,16 @@ define('OG_MEMBERSHIP_REQUEST_FIELD', 'og_membership_request');
  */
 
 /**
+* Implements hook_entity_field_storage_info_alter().
+*/
+function og_entity_field_storage_info_alter(&$fields, \Drupal\Core\Entity\EntityTypeInterface $entity_type) {
+  if (!empty($fields[OG_AUDIENCE_FIELD])) {
+    // Decorate the audience field instance to set the custom storage.
+    $fields[OG_AUDIENCE_FIELD] = new \Drupal\og\Field\FieldStorageConfig($fields[OG_AUDIENCE_FIELD]);
```

Comment:
So we basically switch to `FieldStorageConfig` just so we could disable saving the storage?

I'd be happy to keep this as a plan b approach, and try to push a "fix" to D8. What do you think?